README.md

```diff
@@ -787,32 +812,32 @@ Now the prover creates witness for the commitment `Statement` using the randomne
 const committeds = [blinding];
 for (const i of blindedIndices) {
   // The messages are encoded before committing
-  committeds.push(Signature.encodeMessageForSigning(blindedMessages.get(i)));
+  committeds.push(BBSSignature.encodeMessageForSigning(blindedMessages.get(i)));
 }
 const witness1 = Witness.pedersenCommitment(committeds);
 const witnesses = new Witnesses();
 witnesses.add(witness1);
 
-const proof = CompositeProofG1.generate(proofSpec, witnesses);
+const proof = CompositeProof.generate(proofSpec, witnesses);
 ```
 
 Signer now verifies the proof. Note that the signer independently creates the `ProofSpec` as he knows which attributes are being 
 hidden from him. If the proof is correct, signer creates a blind signature using the known attributes and the commitment 
 and sends to the prover.
 
 ```ts
-expect(proof.verify(proofSpec).verified).toEqual(true);
+proof.verify(proofSpec).verified;  // true
 
 // Signer is convinced that user knows the opening to the commitment
 
 // Signer creates a blind signature with these revealed messages and the commitment.
 revealedMessages.set(1, stringToBytes('John Smith'));
 revealedMessages.set(3, stringToBytes('john.smith@emample.com'));
 revealedMessages.set(4, stringToBytes('New York'));
-const blindSig = BlindSignatureG1.generate(request.commitment, revealedMessages, sk, params, true);
+const blindSig = BBSBlindSignature.generate(request.commitment, revealedMessages, sk, params, true);
 ```
 
-The prover can now "unblind" the signature meaning he can convert a blind signature into a regular BBS+ signature 
+The prover can now "unblind" the signature meaning he can convert a blind signature into a regular BBS signature 
```

Comment:
Unblinding does not apply to BBS. See [this](https://github.com/docknetwork/crypto-wasm-ts/blob/master/tests/composite-proofs/social-kyc.spec.ts#L134).